openshift/templates/nginx-runtime/Dockerfile

```diff
@@ -10,21 +10,9 @@ RUN mkdir -p /etc/nginx/
 RUN echo "" > /etc/nginx/publicServerEnvironmentSettings.js
 RUN mkdir -p /tmp/app/dist/
 RUN ln -sf /etc/nginx/publicServerEnvironmentSettings.js /tmp/app/dist/publicServerEnvironmentSettings.js
-# =================================================================================
-# Fix up permissions
-# ref: https://torstenwalter.de/openshift/nginx/2017/08/04/nginx-on-openshift.html
-# - S2I sripts must be executable
-# - Make sure nginx can read and write it's working directories.
-# - The container dynamically configures nginx on startup
-# - The application artifacts live in /tmp
-# ---------------------------------------------------------------------------------
-RUN chmod -R g+rwx $STI_SCRIPTS_PATH
-RUN chmod g+rw /var/cache/nginx \ 
-               /var/run \
-               /var/log/nginx \
-               /etc/nginx/nginx.conf \
-               /tmp
-# =================================================================================
+# changed /etc to /etc/nginx as error thrown from ibm run cloud on this line
+# /etc/nginx was seen to require chmod 777 much later in deployment
+RUN chmod -R 0777 /tmp /var /etc/nginx /mnt /usr/libexec/s2i/
```

Comment:
Just confirming that the intention is to open full permissions on all of those directories?